src/frontend/screens/Login/components/Runner/index.tsx

```diff
@@ -26,40 +26,54 @@ export default function Runner(props: RunnerProps) {
     //window.localStorage.clear()
   }
   return (
-    <div className={`runnerWrapper ${props.class}`}>
-      <div>{props.icon()}</div>
-      {props.isLoggedIn && (
-        <div className="userData">
-          <span>
-            {String(props.user).slice(0, maxNameLength) +
-              (String(props.user).length > maxNameLength ? '...' : '')}
-          </span>
-        </div>
-      )}
-      <div>
-        {!props.isLoggedIn ? (
-          <Link to={props.loginUrl}>
-            <div className="runnerLogin">{t('button.login', 'Login')}</div>
-          </Link>
-        ) : (
-          <div
-            className="runnerLogin logged"
-            onClick={() => {
-              handleLogout()
-            }}
-          >
-            {t('userselector.logout', 'Logout')}
+    <>
+      <div className={`runnerWrapper ${props.class}`}>
+        <div className={`runnerIcon ${props.class}`}>{props.icon()}</div>
+        {props.isLoggedIn && (
+          <div className="userData">
+            <span>
+              {String(props.user).slice(0, maxNameLength) +
+                (String(props.user).length > maxNameLength ? '...' : '')}
+            </span>
           </div>
         )}
+        <div className="runnerButtons">
+          {!props.isLoggedIn ? (
+            <Link to={props.loginUrl}>
+              <div className="runnerLogin">{`${
```

Comment:
we don't really need this extra div here, we can apply the same style to the `.runnerButtons a` selector instead and we can remove the div

Reply:
Good call, I've just moved the classname to the Link and removed the added div